entities/dashboard/staffWidgetsBoard.json

```diff
@@ -0,0 +1,115 @@
+{
+    "type": "object",
+    "properties": {
+      "uid": {
+        "type": "string",
+        "description": "Unique identifier of the staffWidgetsBoard object"
+      },
+      "board_layout": {
+        "type": "string",
+        "description": "The CSS class of the layout. This will determine the layout of the widgets dashboard based on the class.",
+        "enum": [
+          "dashboard"
+        ]
+      },
+      "type": {
+        "type": "string",
+        "description": "the grid frame of the layout, determines the layout frame containing the board",
+        "enum": [
+          "2*2",
+          "3*3",
+          "4*4",
+          "2*3",
+          "3*2",
+          "2*4",
+          "4*2",
+          "3*4",
+          "4*3"
+        ]
+      },
+      "sections": {
+        "description": "An array of dashboard sections. Each section contains a set of properties defining it and a list of widgets that will be populated into the section once it is displayed",
+        "type": "array",
+        "items":{
+          "code_name": {
+            "type": "string",
+            "description": "a code name represent type of the section",
+            "enum": [
+              "TopRight",
+              "TopLeft",
+              "Top",
+              "BottomRight",
+              "Bottom,Left",
+              "Bottom"
+            ]
+          },
+          "widgets": {
+            "description": "An array of widgets uids which will be populated in the section",
+            "type": "array",
+            "items": {
+              "uid": {
```

Comment:
make this a reference to the widget entity

Reply:
It's not same entity of the widget, it's only uid and dimensions

Reply:
that's not the design. why?